tests/test.py

```diff
@@ -51,16 +57,25 @@ def _require_package(pkg_name):
     installed_pkgs = dcos.package.installed_packages(pkg_manager, None, None, False)
     if not any(pkg['name'] == pkg_name for pkg in installed_pkgs):
         shakedown.install_package(pkg_name, wait_for_completion=True)
-    shakedown.wait_for(_is_hdfs_ready, ignore_exceptions=False, timeout_seconds=600)
+    shakedown.wait_for(_is_hdfs_ready, ignore_exceptions=False, timeout_seconds=900)
 
 
-DEFAULT_HDFS_TASK_COUNT=8
+DEFAULT_HDFS_TASK_COUNT=10
 def _is_hdfs_ready(expected_tasks = DEFAULT_HDFS_TASK_COUNT):
     running_tasks = [t for t in shakedown.get_service_tasks('hdfs') \
                      if t['state'] == 'TASK_RUNNING']
     return len(running_tasks) >= expected_tasks
 
 
+def _run_janitor(service_name):
```

Comment:
Is this not already in shakedown?  Wasn't nick putting this in shakedown?  If not, we should.